README.md

```diff
@@ -1,12 +1,5 @@
 # Jaeger website
 
-[doc-img]: https://readthedocs.org/projects/jaeger/badge/?version=latest
-[doc]: http://jaeger.readthedocs.org/en/latest/
-[project]: https://readthedocs.org/projects/jaeger/
-[ci-img]: https://travis-ci.org/jaegertracing/documentation.svg?branch=master
-[ci]: https://travis-ci.org/jaegertracing/documentation
-[jaeger]: https://jaegertracing.io/
-
```

Comment:
L7-12 not needed anymore?

Reply:
This doesn't actually show up on the [rendered README](https://github.com/jaegertracing/documentation) so I assumed that they're superfluous. If these lines serve some purpose I'd be happy to add them back.

Reply:
are you referring to the links above?  They are just shortcuts, don't seem to be used anymore.